spring-xd-messagebus-kafka/src/main/java/org/springframework/xd/dirt/integration/kafka/KafkaMessageBus.java

```diff
@@ -156,6 +156,8 @@ public Object deserialize(byte[] bytes) throws ZkMarshallingError {
 
 	private static final String XD_REPLY_CHANNEL = "xdReplyChannel";
 
+	private static final String XD_HISTORY = "xdHistory";
```

Comment:
Should this be public somewhere?  Similar to xdReplyChannel as well, just never noticed it before.